app/src/main/java/com/kickstarter/viewmodels/ProjectCardHolderViewModel.java

```diff
@@ -0,0 +1,279 @@
+import com.kickstarter.models.Category;
+import com.kickstarter.models.Photo;
+import com.kickstarter.models.Project;
+import com.kickstarter.models.User;
+import com.kickstarter.ui.viewholders.ProjectCardViewHolder;
+
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.coalesce;
+
+public interface ProjectCardHolderViewModel {
+
+  interface Inputs {
+    /** Call to configure the view model with a project. */
+    void configureWith(Project project);
+    void projectClicked();
+  }
+
+  interface Outputs {
+    /** Emits the string name of the projects category. */
+    Observable<String> backersCountText();
+    Observable<Boolean> backingViewGroupIsGone();
+    Observable<String> categoryNameText();
+    Observable<String> deadlineCountdownText();
+    Observable<Boolean> featuredViewGroupIsGone();
+    Observable<List<User>> friendsForNamepile();
+    Observable<String> friendAvatarUrl();
+    Observable<Boolean> imageIsInvisible();
+    Observable<Boolean> friendBackingViewIsHidden();
+    Observable<Boolean> fundingUnsuccessfulTextViewIsGone();
+    Observable<Boolean> metadataViewGroupIsGone();
+    Observable<Project> projectOutput();
+    Observable<Integer> percentageFunded();
+    Observable<Boolean> percentageFundedProgressBarIsGone();
+    Observable<String> percentageFundedText();
+    Observable<String> photoUrl();
+    Observable<String> blurbText();
+    Observable<String> nameText();
+    Observable<Project> notifyDelegateOfProjectClick();
+    Observable<Boolean> potdViewGroupIsGone();
+    Observable<DateTime> projectCanceledAt();
+    Observable<DateTime> projectFailedAt();
+    Observable<Boolean> projectStateViewGroupIsGone();
+    Observable<DateTime> projectSuccessfulAt();
+    Observable<DateTime> projectSuspendedAt();
+    Observable<String> rootCategoryName();
+    Observable<Void> setDefaultTopPadding();
+    Observable<Void> setMetadataTopPadding();
+    Observable<Boolean> starredViewGroupIsGone();
+    Observable<Boolean> successfullyFundedTextViewIsGone();
+  }
+
+  final class ViewModel extends ActivityViewModel<ProjectCardViewHolder> implements Inputs, Outputs {
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      final Observable<Boolean> projectInEndedStates = this.project
+        .map(
+          p -> p.state().equals(Project.STATE_SUCCESSFUL)
+            || p.state().equals(Project.STATE_CANCELED)
+            || p.state().equals(Project.STATE_FAILED)
+            || p.state().equals(Project.STATE_SUSPENDED)
+        );
+
+      this.projectOutput = this.project;
+
+      this.backersCountText = this.project
+        .map(Project::backersCount)
+        .map(NumberUtils::format);
+
+      this.backingViewGroupIsGone = this.project
+        .map(p -> p.isBacking() || p.isStarred() || p.isPotdToday() || p.isFeaturedToday());
```

Comment:
how does `backingViewGroup` compare to `metadataViewGroup`? just tryna understand what is being shown/hidden...

_edit_: ohhhh I think I understand! this is the backing view group _inside_ the metadata group? perhaps we could rename some stuff to make that clearer?

with that clarity, I now wonder if this output is correct. should it not be gone just when not backing? I'll look at the tests to see how it all comes out in the end...